### DISCUSSION TITLE
importing styled doesn't exist

### DISCUSSION THREAD

**Discussion by rubenmonterogonzalez** [chakra-ui/panda#892] [Q&A] [answered]
### Description

```import { styled } from '@/styled-system/jsx'```

After project set-up, I'm getting the error, Module '"@/styled-system/jsx"' has no exported member 'styled'.ts(2305)

I'm not seeing it created in the styled-system folder.

This is from docs page https://panda-css.com/docs/concepts/style-props

### Link to Reproduction

no styled in codegen panda output

### Steps to reproduce

in the base project setup, run codegen panda. There's no styled exported

### Panda CSS Version

0.5.0

### Browser

Google Chrome Version 114.0.5735.199

### Operating System

- [ ] macOS
- [X] Windows
- [ ] Linux

### Additional Information

Following the docs here: https://panda-css.com/docs/concepts/style-props

**Comment by astahmer** [comment]
you need to define the `jsxFramework` so that Panda knows how `styled` should be generated according to the different kind of JSX there is

https://panda-css.com/docs/concepts/style-props#configure-jsx

**Comment by rubenmonterogonzalez** [comment]
This is done on panda.config.
```export default defineConfig({
  // ...
  jsxFramework: 'react'
})```

**Comment by astahmer** [comment]
can you provide a minimal repro from this template ? https://github.com/astahmer/panda-vite-template

**Comment by minhchu** [comment, marked as answer]
After adding `  jsxFramework: 'react'` in panda.config you should run `panda codegen` again

**Comment by rubenmonterogonzalez** [comment]
I created a repo: https://github.com/rubenmonterogonzalez/next-with-panda

Now it fails this line: ```import { panda } from '@/styled-system/jsx'``` on page.tsx.

If I active this line in panda.config.ts : ``` // jsxFactory: 'panda', *Here fails panda ""import { panda } from '@/styled-system/jsx'""``` and execute codegen, then styled fails ```import { styled } from '@/styled-system/jsx'```

**Reply by minhchu** [reply]
I think it works as expected. Since you enable `jsxFactory: 'panda'` it will rename `styled` to `panda` so `import { styled } from '@/styled-system/jsx'` does not exist

**Reply by astahmer** [reply]
yeah this is the issue. defaults factory name is `styled`, if you specify to panda then you should import it as `import { panda } from '@/styled-system/jsx'`

**Reply by rubenmonterogonzalez** [reply]
Ok, understood then.
Thanks for yur help guys!
